e2e/pages/confirmService/uploadDocuments.page.js

```diff
@@ -0,0 +1,31 @@
+const {I} = inject();
+
+module.exports = {
+
+  fields: {
+    servedDocumentFiles: {
+      id: '#servedDocumentFiles_servedDocumentFiles',
+      options: [
+        '#servedDocumentFiles_particularsOfClaim',
+        '#servedDocumentFiles_medicalReports',
+        '#servedDocumentFiles_scheduleOfLoss',
+        '#servedDocumentFiles_certificateOfSuitability',
+        '#servedDocumentFiles_other'
+      ]
+    }
+  },
+
+  async uploadServedDocuments(file) {
+    I.waitForElement(this.fields.servedDocumentFiles.id);
+
+    for (const fileType of this.fields.servedDocumentFiles.options) {
```

Comment:
e2e is currently failing in pipeline within this loop, any ideas appreciated 😅

Reply:
https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_UCMC%2Fucmc-service/detail/PR-21/11/tests/

https://build.platform.hmcts.net/job/HMCTS_UCMC/job/ucmc-service/job/PR-21/11/artifact/output/Solicitor_confirms_service.failed.png

```
Error
Element "Field" was not found by text|CSS|XPath
Stacktrace
Element "Field" was not found by text|CSS|XPath
```